Cargo.lock

```diff
@@ -5834,18 +5797,17 @@ dependencies = [
 
 [[package]]
 name = "tokio"
-version = "1.12.0"
+version = "1.16.1"
```

Comment:
I find [tokio 1.17](https://github.com/tokio-rs/tokio/releases/tag/tokio-1.17.0) has been released with bug fixes and performance improvements. Any consideration about not using the latest?

Reply:
All I do is `cargo update -p tokio`, seems it's not recognized by cargo.